Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+pipeline {
+  agent {
+    kubernetes {
+      label 'package-linux'
+      yamlFile 'KubernetesPod.yaml'
+      workingDir '/home/jenkins/agent'
+    }
+  }
+
+  options {
+    disableConcurrentBuilds(abortPrevious: true)
+    buildDiscarder logRotator(numToKeepStr: '15') // Retain only last 15 builds to reduce space requirements
```

Comment:
@MarkEWaite any thoughts on whether this is appropriate or too many to keep?

It's 240mb per build of artifacts

Reply:
I'd limit the history to 5 builds.  I don't know why we would refer to more than 5 builds from history and the space savings will help avoid file system cleanup like was needed last week

Reply:
> I'd limit the history to 5 builds.

Done